### DISCUSSION TITLE
is the website brython.info is GITted ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2485] [Ideas]
Hi,

I often go in the documentation page of [brython.info](https://brython.info) but I see a lot of room for improvement or additional content..

I was wondering if those page are forkable/pullable :) under a GIT system (or alike) because I would like to propose some content.

If it's not the case, maybe it's an option to look for ?!

**Comment by PierreQuentel** [comment]
Hi,

The HTML pages at address _brython.info/static_doc_ are not in the GIT repository, but they are generated from the pages in folder **`/doc`** of the repo.

These pages are written with the Markdown syntax, you can edit them and run the program **`/scripts/make_doc.py`** to generate a new version of folder **`/static_doc/<version>`** locally, where **`<version>`** is the current major version (3.12 at time of writing).

**Comment by denis-migdal** [comment]
Hi ;)


I made some git issues about improvement that could be done in the doc, don't hesitate to get inspirations from it ;)
